templates/docs/vehicle/silence_1.yaml

```diff
@@ -20,4 +20,5 @@ render:
       vin: W... # Erforderlich, wenn mehrere Fahrzeuge des Herstellers vorhanden sind # Optional
       capacity: 50 # Akku-Kapazität in kWh # Optional
       phases: 3 # Die maximale Anzahl der Phasen welche genutzt werden können # Optional
+      icon: # Optional
```

Comment:
sollten wir hier nicht Bike oder Scooter vorgeben?

Reply:
Niu fehlt auch als Scooter

Reply:
Guter Punkt. Die Scooter Defaults sind drin. Bei den beiden Scooter-Typen die wir haben (Niu, Silence) sind das jetzt als nicht-advanced mit default drin. Damit wird's durch `configure` bei Neueinrichtungen in die Config geschrieben und taucht auch in der Doku auf.

Alternativ könnten wir die Defaults auch in den jeweiligen Klassen hinterlegen. Dann würde es auch rückwirkend greifen. Da es bislang aber aber nur die beiden haben ist das so mMn ausreichend.